internal/app/init/main.go

```diff
@@ -192,20 +194,44 @@ func root() (err error) {
 	svcs := system.Services(data)
 	defer svcs.Shutdown()
 
+	// Instantiate internal init API
+	api := reg.NewRegistrator(data)
+	server := factory.NewServer(api)
+	listener, err := factory.NewListener(
+		factory.Network("unix"),
+		factory.SocketPath(constants.InitSocketPath),
+	)
+	if err != nil {
+		panic(err)
+	}
+	defer server.Stop()
+
+	go func() {
+		// nolint: errcheck
+		server.Serve(listener)
+	}()
+
 	// Start containerd.
 	svcs.Start(&services.Containerd{})
 
 	go startSystemServices(startupErrCh, data)
 	go startKubernetesServices(startupErrCh, data)
 
 	select {
+	case <-api.ShutdownCh:
+		log.Printf("Poweroff via API received")
```

Comment:
Can we change these logs to all lowercase? Its bit of a nit, but the rest of the codebase is all lowercase. I don't have a huge preference for all lowercase, but I think consistency is nice.

Reply:
sure, will do!

Reply:
updated!